core/src/main/scala/io/odin/formatter/options/ThrowableFormat.scala

```diff
@@ -18,4 +25,15 @@ object ThrowableFormat {
     final case class Fixed(size: Int) extends Indent
   }
 
+  sealed trait Filter
+  object Filter {
+    final case object NoFilter extends Filter
+    final case class Excluding(prefixes: Set[String]) extends Filter
+
+    object Excluding {
+      def apply(prefixes: String*): Excluding = new Excluding(prefixes.toSet)
+      def apply(prefixes: IterableOnceCompat[String]): Excluding = new Excluding(prefixes.toSetCompat)
```

Comment:
What's the point of going so generic here? It adds unneccessary complexity for benefits I'm clearly missing

Reply:
I think, users may want to construct `ThrowableFormat.Filter.Excluding` from any `Traversable` (`Iterable`)
e.g. read from typesafe config and create logging format, having to do something like this:
`Filter.Excluding(config.getStringList("exclude"): _ *)`
or
`Filter.Excluding(config.getStringList("exclude").toSet)`

Reply:
I think it's fine for the end user to convert whatever they have to the set on their own